data_services/deploy/ansible/Vagrantfile

```diff
@@ -0,0 +1,22 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+Vagrant.configure("2") do |config|
+  config.vm.box = "trusty64server"
+  config.vm.box_url = "https://oss-binaries.phusionpassenger.com/vagrant/boxes/latest/ubuntu-14.04-amd64-vbox.box"
+  config.ssh.username = 'vagrant'
+  config.ssh.forward_agent = true
+
+  config.vm.define :production do |production|
+  	production.vm.network :public_network, :bridge => 'eth0', :auto_config => false
```

Comment:
@aashish,

Currently the site on the VM is reached by going to http://localhost:8000 <http://localhost:8000/>, but the thumbnail URL's are generated as SITE URL + MEDIA_URL + /"thumbs/<thumbnail_name>.png --> http://127.0.0.1/uploaded/thumbs/<thumbnail_name>.png <http://127.0.0.1/uploaded/thumbs/%3Cthumbnail_name%3E.png> - which cannot be accessed.

By using a private network IP and setting SITEURL to that same IP, this problem is avoided because you can access the site at for example http://192.168.33.12 <http://192.168.33.12/> and the thumbnail URL will be generated as http://192.168.33.12/uploaded/thumbs/<thumbnail_name>.png <http://192.168.33.12/uploaded/thumbs/%3Cthumbnail_name%3E.png>



> On Sep 10, 2015, at 10:14 AM, Aashish Chaudhary <notifications@github.com> wrote:
> 
> In data_services/deploy/ansible/Vagrantfile <https://github.com/Kitware/minerva/pull/98#discussion_r39164306>:
> 
> > @@ -0,0 +1,22 @@
> > +# -*- mode: ruby -*-
> > +# vi: set ft=ruby :
> > +Vagrant.configure("2") do |config|
> > +  config.vm.box = "trusty64server"
> > +  config.vm.box_url = "https://oss-binaries.phusionpassenger.com/vagrant/boxes/latest/ubuntu-14.04-amd64-vbox.box"
> > +  config.ssh.username = 'vagrant'
> > +  config.ssh.forward_agent = true
> > +
> > +  config.vm.define :production do |production|
> > +  	production.vm.network :public_network, :bridge => 'eth0', :auto_config => false
> Thanks @mbertrand <https://github.com/mbertrand>. How does it fixes the thumbnail problem? I will try this change but just curious.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Kitware/minerva/pull/98/files#r39164306>.
>